### PR TITLE
Updated working in pairs rule

### PR DESCRIPTION
**Pull request by SamBWagner**
## Notes

*Checked By Calum S and Brady S*

- De-gendered text
- Removed Dead survey from bottom of Rule (will re-enter when I hear back from Warwick)
- Made some lines clearer
- Added a point about asking for help when working alone with some reasoning
- Updated collaboration video with one from the Visual Studio Code team
- Updated the section below the VS Code Live Share video with updated suggestions and links
- Added a Code With Me by JetBrains mention as another tool which can be used for Pair Programming

### CODE REVIEW

File: rules/efficiency-do-you-always-try-to-work-in-pairs/rule.md [hunk 1]
```diff
@@ -24,30 +24,36 @@ For everyone:
 
 * Less time stuck on a problem - you have someone familiar with the project to help you work through the problem
 * You can keep each other motivated  and you absorb knowledge from each other
-* Experience shows that **people** are more productive
+* Experience shows that **people** are more productive than any single person
```
Comment (bradystroud) [inline rules/efficiency-do-you-always-try-to-work-in-pairs/rule.md:27]: This wording is confusing. I don't know I would reword it though.

**Review by bradystroud — APPROVED** [review]

File: rules/efficiency-do-you-always-try-to-work-in-pairs/rule.md (hunk 1 above)
Comment (calumjs) [reply rules/efficiency-do-you-always-try-to-work-in-pairs/rule.md:27]: Agreed a little confusing - how about...
Experience shows that two people working on a problem together are often more productive than one person working alone for twice as long.

Comment (SamBWagner) [reply rules/efficiency-do-you-always-try-to-work-in-pairs/rule.md]: I've added that line instead of the original. Much better 💪

**Review by calumjs — APPROVED** [review]
🛳

**Comment by SamBWagner** [comment]
Waiting for a Piers review before Sqeeeeeeerging.

**Review by UlyssesMaclaren — APPROVED** [review]
